src/ClientResolver.php

```diff
@@ -327,7 +341,7 @@ class ClientResolver
             'valid' => ['array', 'string'],
```

Comment:
The configuration resolver technically doesn't handle arrays (nor should it), so I think we'll need to move 'string' to the front here.

Reply:
Good catch. Thanks!